404.html

```diff
@@ -0,0 +1,20 @@
+---
+---
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    {% include meta.html %}
+    <title>Hierplane - Error</title>
```

Comment:
Maybe `Page Not Found - Hierplane`?  I usually start with the more specific title.  Also `Error` sounds ominous.

Reply:
Sure, no prob. Will change.

Reply:
Done in latest commit.